src/libaktualizr/utilities/utils.cc

```diff
@@ -457,7 +457,7 @@ std::string Utils::randomUuid() {
   return boost::uuids::to_string(uuid_gen());
 }
 
-// FIXME Doesn't work with broken symlinks
+// TODO: Doesn't work with broken symlinks
```

Comment:
Maybe we do not need to have that as a `TODO` if we don't need it, but we definitely should document it in a comment somewhere.

Reply:
Good point. I'll remove the TODO and leave the comment. No need to make it sound so scary and important.